Max/src/main/java/springbootApp/app/controllers/InstrumentoFinancieroController.java

```diff
@@ -0,0 +1,64 @@
+package  springbootApp.app.controllers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import  springbootApp.app.controllers.mappers.InstrumentoMapper;
+import  springbootApp.app.DTOs.InstrumentoDTO;
+import  springbootApp.app.models.InstrumentoFinanciero;
+import  springbootApp.app.services.InstrumentoFinancieroService;
+
+@RestController
+@RequestMapping("/instrumento-Financiero")
+public class InstrumentoFinancieroController {
+
+    private static final Logger log = LoggerFactory.getLogger(InstrumentoFinancieroController.class);
+    @Autowired
+    private InstrumentoFinancieroService instrumentoFinancieroService;
+
+
+    @PostMapping("/")
+    public ResponseEntity<?> registrar(@RequestBody InstrumentoDTO instrumentoDTO) throws Exception{
+            log.info("registrando instrumento...");
+            InstrumentoFinanciero instrumento = InstrumentoMapper.instrumentoDTOToInstrumento(instrumentoDTO);
+            instrumentoFinancieroService.registrarInstrumentoFinanciero(instrumento);
```

Comment:
la variable "instrumento" puede ser null, por lo que recomendaria validarlo antes de enviarlo como parametro de entrada a otro metodo.

```suggestion
    public ResponseEntity<?> registrar(@RequestBody InstrumentoDTO instrumentoDTO) throws Exception{
            log.info("registrando instrumento...");
            InstrumentoFinanciero instrumento = InstrumentoMapper.instrumentoDTOToInstrumento(instrumentoDTO);
            if (instrumento == null)
             retornar error
            instrumentoFinancieroService.registrarInstrumentoFinanciero(instrumento);
```